palisade-service/mvn_dependency_tree.txt

```diff
@@ -248,7 +246,7 @@ org.antlr/stringtemplate/version = 3.2.1
 
 org.apache.avro/avro/version = 1.8.2
 
-org.apache.commons/commons-compress/version = 1.18
+org.apache.commons/commons-compress/version = 1.8.1
```

Comment:
Still looks like we're going back to version 1.8.1 for commons-compress - can you hard code this version as well please???